templates/versions.interface.hbs

```diff
@@ -22,11 +22,17 @@
       <td>{{columnsCount}}</td>
         <td>
           {{#if hasEntries}}
-            <ul>
-              <li><a href="#" data-version-preview="{{id}}">Preview data</a></li>
-              <li><a href="#" data-version-restore="{{id}}">Restore</a></li>
-              <li><a href="#" data-version-copy="{{id}}">Copy to new data source</a></li>
-            </ul>
+            <div class="btn-group" role="group">
+              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Actions
+                <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu">
```

Comment:
With the example on https://getbootstrap.com/docs/3.4/javascript/#dropdowns, if you add the following styles to the `UL`, it'll align the dropdown to the right edge. You won't need to set the actions column with a fixed width. This assumes `.btn-group` has `position: relative`.

```css
    left: auto;
    right: 0;
```

Reply:
thanks, works 
![image](https://user-images.githubusercontent.com/574210/78057425-ff0bd980-7386-11ea-8b3c-9a62eec85c37.png)